long-term-storage.md

```diff
@@ -59,61 +60,69 @@ $ helm install kubecost/cost-analyzer \
 
 Your deployment should now have Thanos enabled!
 
-> Note: the `thanos-store` pod is by default configured to request 2 Gb in memory.
+> Note: If you need to delete a previous install of kubecost, do this before creating the secret in step 2
+
+> Note: The `thanos-store` pod is by default configured to request 2 Gb in memory.
 
 <a name="verify-thanos"></a>
 **Verify Installation**  
 In order to verify a correct installation, start by ensuring all pods are running without issue. If the pods mentioned above are not running successfully, then view pod logs for more detail. A common error is as follows, which means you do not have the correct access to the supplied bucket:
 
-```
+```text
 thanos-svc-account@project-227514.iam.gserviceaccount.com does not have storage.objects.list access to thanos-bucket., forbidden"
 ```
 
 Assuming pods are running, use port forwarding to connect to the `thanos-query-http` endpoint:
+
+```shell
+kubectl port-forward svc/kubecost-thanos-query-http 8080:10902 --namespace kubecost
 ```
-$ kubectl port-forward svc/kubecost-thanos-query-http 8080:10902 --namespace kubecost
-```
+
 Then navigate to http://localhost:8080 in your browser. This page should look very similar to the Prometheus console.
 
-![image](https://user-images.githubusercontent.com/334480/66616984-1076e480-eba1-11e9-8dd2-7c20541ad0b1.png)
+![image](https://raw.githubusercontent.com/kubecost/docs/main/images/thanos-query.png)
```

Comment:
I'm showing these images as missing in GH preview -- https://github.com/kubecost/docs/blob/82c75474bff957c06fcb366540f85413b0a8641c/long-term-storage.md

Reply:
That's because they are direct paths to their location in `main` once merged. Per the syle guide @kbrwn put together, image paths must be absolute, not relative. I also verified with him to be sure as it makes testing difficult (which you discovered).